include/mbedtls/ccm_default.h

```diff
@@ -0,0 +1,140 @@
+/**
+ * \file ccm_default.h
+ *
+ * \brief Default function definitions for CCM
+ *
+ * \note These functions are meant for the alternative implementors,
+ *       in case a need for a fallback to the default implementation
+ *       is required. Application developers should call the
+ *       base function instead (without the _default suffix).
+ *
+ *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
```

Comment:
This is a new file. Should we update the copyright date?

Reply:
Good catch. We should definitely update the date